tests/test_commands.py

```diff
@@ -446,6 +447,20 @@ def test_linesort(runner, lines):
     assert data.pen_up_length == 0
 
 
+@pytest.mark.parametrize(
+    "lines",
+    [
+        " ".join(s)
+        for s in itertools.permutations(
+            ["line 0 0 0 10", "line 0 10 10 10", "line 0 0 10 0", "line 10 0 10 10"]
+        )
+    ],
+)
+def test_lineshuffle(runner, lines):
+    res = runner.invoke(cli, f"{lines} lineshuffle dbsample dbdump")
+    assert res.exit_code == 0
+
```

Comment:
I'm not sure I fully understand what this test attempts to do. It basically checks that the command doesn't just fail? Adding `Command("lineshuffle")` above does basically the same, and give how simple this command is, I'd just remove this test altogether.

Reply:
Yeah, that's basically all I could think of a for a test.  I threw this in as a stub for conversation -- if you had any better ideas of a test for this -- but if you can't think of any better tests I will delete this.